tests/formats/test_bgsnds.py

```diff
@@ -0,0 +1,7 @@
+from tests.test_lib import parse_build_compare_editor
+
+from mercury_engine_data_structures.formats.bgsnds import Bgsnds
+
+
+def test_bgsnds(dread_file_tree):
+    parse_build_compare_editor(Bgsnds, dread_file_tree, "system/snd/guisoundevents.bgsnds")
```

Comment:
Is there really only one file that uses that format?

Reply:
yeah, it's basically for the GUI sounds (switching selected item, accept/decline, etc)